retrieval/target.go

```diff
@@ -359,9 +364,15 @@ func (t *Target) scrape(sampleAppender storage.SampleAppender) (err error) {
 	}()
 
 	for samples := range t.ingestedSamples {
-		for _, s := range samples {
-			s.Metric.MergeFromLabelSet(baseLabels, clientmodel.ExporterLabelPrefix)
-			sampleAppender.Append(s)
+		if t.proxy {
+			for _, s := range samples {
+				sampleAppender.Append(s)
```

Comment:
If we removed the empty instance label completely, then we would get an instance label attached if we export it further up via federation, wouldn't we?